90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -33,7 +33,7 @@ zlog() {
   offset="${#prefix}"
 
   # account for the fzf gutter and kernel timestamp
-  _WIDTH=$(( $( tput cols ) - ( offset - 15 ) ))
+  _WIDTH=$(( COLUMNS - ( offset - 15 ) ))
```

Comment:
Is `$COLUMNS` guaranteed to be defined, or does `tput cols` buy anything extra?

Reply:
I found that `$COLUMNS` is only defined if `bash` is an interactive shell of some kind. If it's not, that variable is undefined. In either case, that's dead code now that I've rebased onto the new logging handler.